CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ set(DOXYGEN_USE_MATHJAX YES)
 # JRL-cmakemodule setup
 include(cmake/base.cmake)
 include(cmake/boost.cmake)
+include(cmake/python.cmake)
```

Comment:
We shouldn't include this file. eigenpy package its version, and this should be the good one to use

Reply:
Not doing it ends up with:
```
--- stderr: dynamic-graph-python
CMake Error at src/CMakeLists.txt:9 (python_install_on_site):
  Unknown CMake command "python_install_on_site".
```

Reply:
with ```eigenpy-3.1.0```

Reply:
Can you try again with 3.1.1 ? I think 3.1.0 was broken on this topic

Reply:
I will remove this line and will use locally. eigenpy is still in 3.1.0 on conda (the environment I am currently testing), and I would like to avoid recompiling pinocchio.